pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1043,103 +1043,6 @@ def is_hermitian(self):
         return True
 
 
-class CZ(Operation):
-    r"""CZ(wires)
-    The controlled-Z operator
-
-    .. math:: CZ = \begin{bmatrix}
-            1 & 0 & 0 & 0 \\
-            0 & 1 & 0 & 0\\
-            0 & 0 & 1 & 0\\
-            0 & 0 & 0 & -1
-        \end{bmatrix}.
-
-    .. note:: The first wire provided corresponds to the **control qubit**.
-
-    **Details:**
-
-    * Number of wires: 2
-    * Number of parameters: 0
-
-    Args:
-        wires (Sequence[int]): the wires the operation acts on
-    """
-    num_wires = 2
-    num_params = 0
-    """int: Number of trainable parameters that the operator depends on."""
-
-    basis = "Z"
-
-    def label(self, decimals=None, base_label=None, cache=None):
-        return base_label or "Z"
-
-    @staticmethod
-    @lru_cache()
-    def compute_matrix():  # pylint: disable=arguments-differ
-        r"""Representation of the operator as a canonical matrix in the computational basis (static method).
-
-        The canonical matrix is the textbook matrix representation that does not consider wires.
-        Implicitly, this assumes that the wires of the operator correspond to the global wire order.
-
-        .. seealso:: :meth:`~.CZ.matrix`
-
-        Returns:
-            ndarray: matrix
-
-        **Example**
-
-        >>> print(qml.CZ.compute_matrix())
-        [[ 1  0  0  0]
-         [ 0  1  0  0]
-         [ 0  0  1  0]
-         [ 0  0  0 -1]]
-        """
-        return np.array([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, -1]])
-
-    @staticmethod
-    def compute_eigvals():  # pylint: disable=arguments-differ
```

Comment:
did you drop this because we want to use Controlled's default behaviour?

Reply:
I think so. Dropping `compute_eigvals` was suggested by @albi3ro so maybe she can give you proper arguments behind this choice :)